source/AMDSBufferGroupInfo.h

```diff
@@ -36,6 +36,8 @@ class AMDSBufferGroupInfo
 
 	void setAxes(const QList<AMDSAxisInfo> &axes) { axes_ = axes; }
 
```

Comment:
Please try to comment as you, it'll make the task way easier than having to comment once you're all done.  Or, the more likely case, it'll be left.